examples/creating_printing_table/creating_printing_table.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015 Steinwurf ApS
+// All Rights Reserved
+//
+// Signed-off-to-Steinwurf-by: Juan Cabrera <jcabre14@student.aau.dk>
+// Distributed under the "BSD License". See the accompanying LICENSE.rst file.
+
+
+#include <fstream>
+#include <cstdint>
+
+#include <tables/table.hpp>
+#include <tables/csv_format.hpp>
+#include <tables/json_format.hpp>
+#include <tables/python_format.hpp>
+
+/// @example creating_printing_table.cpp
+///
+/// Simple example showing how to create a table
+/// and print it in any desired format.
+
+int main()
+{
+
+    // In this example we will generate 100 random numbers. We then
+    // will store those numbers in a table with two columns:
+    // The random number and the iteration in wich it was generated.
+
```

Comment:
I fixed some of the spelling and wording, this should be more correct :)

// In this example we will generate 100 random numbers. We **will**
// **then** store those numbers in a table with two columns:
// The random number and the iteration in **which** it was generated.